### DISCUSSION TITLE
蓝奏云url和cookie挂载失败

### DISCUSSION THREAD

**Discussion by holob** [AlistGo/alist#360] [Q&A] [answered]
阿里云盘、天翼云盘、123盘已经配置完成，可以正常使用。
尝试了很久挂载蓝奏云一直失败，两种方式都测试了，可以正确读取文件夹和文件目录，但是无法下载文件。
实在找不到解决方法，在此向您提问
日志：
![微信截图_20220114225533](https://user-images.githubusercontent.com/94737046/149536346-594c4812-2b1b-4eb6-85d8-06bf201173e3.png)
报错：
![微信截图_20220114225551](https://user-images.githubusercontent.com/94737046/149536401-60d03c40-21f8-41f0-ae18-db34bb648ea6.png)

**Comment by xhofe** [comment]
发一个你的蓝奏云的URL和密码。
不想公开的话可以通过邮件：support@nn.ci

**Comment by holob** [comment, marked as answer]
> 发一个你的蓝奏云的URL和密码。
> 
> 不想公开的话可以通过邮件：support@nn.ci

感谢回复
下面是我的蓝奏云文件分享链接：
https://wwb.lanzouw.com/b02uokckj 
密码:1spv

**Reply by xhofe** [reply]
应该是蓝奏云不同用户域名不同的问题，将在下个版本修复。

**Reply by holob** [reply]
> 应该是蓝奏云不同用户域名不同的问题，将在下个版本修复。

感谢回复
期待更新😁
